scripts/lib.sh

```diff
@@ -8,6 +8,9 @@ set -e
 
 GO_AGENT_PKG=${GO_AGENT_PKG:-github.com/kata-containers/agent}
 GO_RUNTIME_PKG=${GO_RUNTIME_PKG:-github.com/kata-containers/runtime}
+RUST_AGENT_PKG=${RUST_AGENT_PKG:-github.com/kata-containers/kata-rust-agent}
+CMAKE_VERSION=${CMAKE_VERSION:-"3.15.3"}
+MUSL_VERSION=${MUSL_VERSION:-"1.1.23"}
```

Comment:
These variables are also defined in `rootfs-builder/rootfs.sh` but as mentioned above do we even need these variables if we determine the values dynamically?

Reply:
I cannot recall the reason exactly.. Maybe it is introduced in intermediate development. Let's remove it to see if the script works

Reply:
Yes, we do need them as GO_VERSION is needed.

Remove it from lib.sh cause test_images.sh fail, we need to keep it.

Since we don't echo the version string from detect_xxx_version nor export  it